contrib/endpoints/src/api_manager/auth/service_account_token.h

```diff
@@ -65,6 +65,7 @@ class ServiceAccountToken {
     JWT_TOKEN_FOR_SERVICE_CONTROL = 0,
     JWT_TOKEN_FOR_CLOUD_TRACING,
     JWT_TOKEN_FOR_FIREBASE,
+    JWT_TOKEN_FOR_AUTHORIZATION_SERVICE,
```

Comment:
Let's add a comment for this one, like "JWT token for accessing the http endpoints that user defines in Firebase Rules".